templates/data/eventing-tools-data.yaml

```diff
@@ -0,0 +1,35 @@
+templates:
+  - from: generic.tmpl
+    render:
+      - to: ../../prow/jobs/eventing-tools/eventing-tools-generic.yaml
+        jobConfigs:
+          - repoName: "github.com/kyma-project/eventing-tools"
+            jobs:
+              - jobConfig:
+                  name: pull-eventing-tools-build
+                  args:
+                    - "--name=eventing-tools"
+                    - "--config=/config/kaniko-build-config.yaml"
+                    - "--context=."
+                    - "--dockerfile=Dockerfile"
+                  run_if_changed: "^.*$"
```

Comment:
If you want to run it on everything, instead of using `run_if_changed`, use `always_run: true`